### DISCUSSION TITLE
How does one determine the current index of the frame that is playing?

### DISCUSSION THREAD

**Discussion by davesargrad** [cookpete/react-player#1819] [Q&A]
Is there a way to know what frame (relative to the start of the HLS stream) is the current frame playing? 

I have a need to know the absolute frame number (starting at the 0'th frame for the very first in the stream, and ending at N-1) where N is the total number of frames in the HLS stream.

Its not the image I want, I know how to get that. Nor is it time that I want. I also know how to get that.

At first glance, this would be a feature similar to "[remotions](https://www.remotion.dev/docs/use-current-frame)" useCurrentFrame hook.

Its a relative index starting at 0, and allowing me to absolutely determine where within the stream I am.

**Comment by hoetmaaiers** [comment]
I am looking for a similar answer. 
@davesargrad , did you become any smarter since October last year?
